spec/index.html

```diff
@@ -393,11 +398,11 @@ <h2>Canonicalization State</h2>
 
     <dl>
       <dt><dfn>blank node to quads map</dfn></dt>
-      <dd>A data structure that maps a <a>blank node identifier</a> to
+      <dd>A <a data-cite="INFRA#ordered-map">map</a> that relates a <a>blank node identifier</a> to
```

Comment:
I think it would be better to say "ordered map" in the text here, even if the link is right:

```suggestion
      <dd>An <a data-cite="INFRA#ordered-map">ordered map</a> that relates a <a>blank node identifier</a> to
```